src/test/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/TestConnectionManager.kt

```diff
@@ -445,8 +444,12 @@ class TestConnectionManager {
                     LOGGER.info { "Rabbit address- ${it.host}:${it.amqpPort}" }
 
                     LOGGER.info { "Restarting the container" }
-                    restartContainer(it)
-                    Thread.sleep(5000)
+                    it.stop()
+                    Thread.sleep(5_000)
```

Comment:
Why do we need a delay between stop and start? As far as I remember the method won't return until the container is finally stopped. The same about `start` method